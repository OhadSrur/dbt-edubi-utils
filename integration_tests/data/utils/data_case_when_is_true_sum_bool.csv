group_label,col_to_check
a,1
a,1
a,0
a,1
a,0