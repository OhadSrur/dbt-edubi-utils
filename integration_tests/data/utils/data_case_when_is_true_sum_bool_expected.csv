group_label,result_expected,result_expected_inverted
a,3,2